ovs/extensions/healthcheck/alba.py

```diff
@@ -195,16 +193,30 @@ def check_if_proxies_work(result_handler):
                     # Encapsulation try for cleanup
                     try:
                         # Generate new namespace name using the preset
-                        namespace_key = 'ovs-healthcheck-ns-{0}'.format(preset_name)
+                        namespace_key = 'ovs-healthcheck-ns-{0}-{1}'.format(preset_name, AlbaHealthCheck.LOCAL_ID)
                         object_key = 'ovs-healthcheck-obj-{0}'.format(str(uuid.uuid4()))
-                        try:
-                            # Create namespace
-                            AlbaCLI.run(command='proxy-create-namespace',
-                                        named_params={'host': ip, 'port': service.ports[0]},
-                                        extra_params=[namespace_key, preset_name])
-                            result_handler.success('Namespace successfully created on proxy {0} with preset {1}!'.format(service.name, preset_name))
-                        except AlbaException:
-                            raise
+                        # Create namespace
+                        AlbaCLI.run(command='proxy-create-namespace',
+                                    named_params={'host': ip, 'port': service.ports[0]},
+                                    extra_params=[namespace_key, preset_name])
+                        # Wai until fully created
+                        namespace_start_time = time.time()
+                        while True:
+                            if time.time() - namespace_start_time > 30:
+                                raise RuntimeError('Creation namespace has timed out after {0}s'.format(time.time() - namespace_start_time))
```

Comment:
Doing it in the if is actually better as it removes the small downtime of allocation the variable and calling for it.